### PR TITLE
Change text for days since installation (EXPOSUREAPP-6574)

### PR DESCRIPTION
**Pull request by chiljamgossow**
New text if installation today or yesterday
Changed calculation to account for date changes (not 24h intervals) based on user time zone

device-2021-04-27-082041

### CODE REVIEW

**Comment by harambasicluka** [comment]
May need to be changes to `2.2.x` as we won't get another translation delivery for `2.1.x`.

**Comment by harambasicluka** [comment]
We'll get a translation delivery, but it is too late for this PR. I have overseen it. Need to ship it with `2.2.x`.

**Review by janetback — APPROVED** [review]
UA reviewed

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/installTime/InstallTimeProvider.kt [hunk 1]
```diff
@@ -1,19 +1,29 @@
...
+        context
+            .packageManager
+            .getPackageInfo(context.packageName, 0)
+            .firstInstallTime
+    )
+        .toLocalDateUserTz()
 
-    val daysSinceInstallation: Long get() = (System.currentTimeMillis() - installTime).roundUpMsToDays()
+    val today: LocalDate
+        get() = Instant.now().toLocalDateUserTz()
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/installTime/InstallTimeProvider.kt:25]: maybe `timeStamper.nowUTC` ?

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/tracing/states/TracingState.kt [hunk 2]
```diff
@@ -131,8 +131,11 @@ data class LowRisk(
     }
 
     fun getDaysSinceInstall(context: Context): String =
-        context.getString(R.string.risk_card_body_days_since_installation)
-            .format(daysSinceInstallation)
+        when (daysSinceInstallation) {
+            0 -> context.getString(R.string.risk_card_body_installation_today)
+            1 -> context.getString(R.string.risk_card_body_installation_yesterday)
+            else -> context.getString(R.string.risk_card_body_days_since_installation).format(daysSinceInstallation)
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/tracing/states/TracingState.kt:137]: `context.getString(R.string.risk_card_body_days_since_installation,daysSinceInstallation)` could format directly ?

**Review by mtwalli — APPROVED** [review]
LGTM , small comments , no blocker for me

**Pull request merged**